src/models/post.ts

```diff
@@ -0,0 +1,16 @@
+const post = (sequelize, Sequelize) => {
+  return sequelize.define('post', {
+    id: {
+      type: Sequelize.UUID,
+      defaultValue: Sequelize.UUIDV4,
+      primaryKey: true,
+    },
+    title: Sequelize.STRING(255),
+    content: Sequelize.STRING(255),
+    chatId: Sequelize.STRING(255),
+    timestamps: true,
+    createdAt: 'date_created',
+    updatedAt: 'date_updated',
+  });
+};
+export = post;
```

Comment:
Should use proper syntax. See [TypeScript Deep Dive](https://basarat.gitbooks.io/typescript/content/docs/project/external-modules.html) for details.